src/Nowruz/modules/general/components/avatarProfile/index.tsx

```diff
@@ -8,53 +8,28 @@ import { Icon } from 'src/Nowruz/general/Icon';
 import css from './avatarProfile.module.scss';
 import { AvatarProfileProps } from './avatarProfile.types';
 
-export const AvatarProfile: React.FC<AvatarProfileProps> = (props) => {
+export const AvatarProfile: React.FC<AvatarProfileProps> = props => {
   const { size, imgUrl, type, verified = false, text, handleClick } = props;
   const verifiedTick = size === 'small' ? smallTick : size === 'medium' ? mediumTick : largeTick;
+  const iconSizes = [
+    { size: 'small', iconSize: 36 },
+    { size: 'medium', iconSize: 48 },
+    { size: 'large', iconSize: 80 },
+  ];
   const getContent = () => {
     if (imgUrl) return <img className={css.imgContainer} src={imgUrl} alt="" />;
     if (text) {
-      if (size === 'small') return <span className={css.textSm}>{text}</span>;
-      if (size === 'medium') return <span className={css.textMd}>{text}</span>;
-      if (size === 'large') return <span className={css.textLg}>{text}</span>;
-    }
-    if (type === 'users') {
-      if (size === 'small')
-        return (
-          <Icon
-            name="user-01"
-            fontSize={36}
-            color={variables.color_grey_600}
-            className="!cursor-pointer"
-            containerClass="cursor-pointer"
-          />
-        );
-      if (size === 'medium')
-        return (
-          <Icon
-            name="user-01"
-            fontSize={48}
-            color={variables.color_grey_600}
-            className="!cursor-pointer"
-            containerClass="cursor-pointer"
-          />
-        );
-      if (size === 'large')
-        return (
-          <Icon
-            name="user-01"
-            fontSize={80}
-            color={variables.color_grey_600}
-            className="!cursor-pointer"
-            containerClass="cursor-pointer"
-          />
-        );
-    }
-    if (type === 'organizations') {
-      if (size === 'small') return <Icon name="building-01" fontSize={36} color={variables.color_grey_600} />;
-      if (size === 'medium') return <Icon name="building-01" fontSize={48} color={variables.color_grey_600} />;
-      if (size === 'large') return <Icon name="building-01" fontSize={80} color={variables.color_grey_600} />;
+      return <span className={css[`text-${size}`]}>{text}</span>;
     }
+    return (
+      <Icon
+        name={type === 'users' ? 'user-01' : 'building-05'}
+        fontSize={iconSizes.find(item => item.size === size)?.iconSize}
```

Comment:
you can create a hash table instead of an array and use `find`
```
const something = {
small: {font: 36, tick: smallTcik},
medium: {font: 48, tick: mediumTick},
large: {font:80, tick: largeTick}
}
something[size].font -> for fontSize
something[size].tick -> for verifiedTick